fission/src/systems/physics/PhysicsSystem.ts

```diff
@@ -719,6 +721,45 @@ class PhysicsSystem extends WorldSystem {
         JOLT.destroy(this._joltBodyInterface)
         JOLT.destroy(this._joltInterface)
     }
+
+    /**
+     * Creates a ghost object and a distance constraint that connects it to the given body
+     * The ghost body is part of the LAYER_GHOST which doesn't interact with any other layer
+     * The caller is responsible for cleaning up the ghost body and the constraint
+     *
+     * @param id The id of the body to be attatched to and moved
+     * @returns The ghost body and the constraint
+     */
+
+    public CreateGodModeBody(id: Jolt.BodyID): [Jolt.Body, Jolt.Constraint] {
+        const body = this.GetBody(id)
+        const bodyPosition = body.GetPosition()
+        const ghostPosition = new THREE.Vector3(bodyPosition.GetX(), bodyPosition.GetY() + 0.05, bodyPosition.GetZ())
+        const ghostBody = this.CreateBox(new THREE.Vector3(0.05, 0.05, 0.05), undefined, ghostPosition, undefined)
+
+        const ghostBodyId = ghostBody.GetID()
+        this._joltBodyInterface.SetObjectLayer(ghostBodyId, LAYER_GHOST)
+        this._joltBodyInterface.AddBody(ghostBodyId, JOLT.EActivation_Activate)
+        this._bodies.push(ghostBodyId)
+
+        const constraintSettings = new JOLT.SwingTwistConstraintSettings()
+        const constraint = constraintSettings.Create(ghostBody, body)
```

Comment:
You don't specify the anchor points for the constraint and it appears to snap to an arbitrary position. We want to make sure that this position is both in compliance with their current configuration (aka. The robot shouldn't be snapping to anywhere, it should just restrict further movement) and easily configurable for when god mode is implemented. It doesn't appear to be determined by the position of the ghost body at all, as I removed it and set the position to zero and it changed very little.